packages/server-wallet/src/wallet/index.ts

```diff
@@ -102,11 +110,13 @@ export class Wallet implements WalletInterface {
     // Bind methods to class instance
     this.getParticipant = this.getParticipant.bind(this);
```

Comment:
Probably doesn't belong in this PR, but it might be time to refactor this code to:
```
const methods = [this.getParticipants, this.updateChannelFunding, ...];
methods.map(m => m.bind(this))
```

Reply:
It would also be nice to have an explanation of why this is necessary.